subcommands/keys/rotate_targets.go

```diff
@@ -60,6 +61,13 @@ func doRotateTargets(cmd *cobra.Command, args []string) {
 	targetid, newCreds := replaceOfflineTargetKey(root, onlineTargetId, creds, keyType)
 	fmt.Println("= New target:", targetid)
 	removeUnusedKeys(root)
+	user, err := api.UserAccessDetails(factory, "self")
+	subcommands.DieNotNil(err)
+	root.Signed.Reason = &client.RootChangeReason{
+		PolisId:   user.PolisId,
+		Message:   "Targets role key rotation",
+		Timestamp: time.Now(),
+	}
```

Comment:
sorry - i missed this. We should probably try and make the CLI consistent with:

 https://github.com/foundriesio/fioctl/commit/b6ca47189c8764f0b335b32eb0530774ba3c1559

Reply:
i.e. - the user can optionally provide a change-reason

Reply:
What would be the help string in this case? Or just us the same as for the root rotation cmd: `"Reason for resigning root.json. Saved in root metadata for tracking change history"`?

Reply:
i think the default reason you have is fine. I'm more saying we need the same flag as that command so that they can do a `--reason "Cause Johnny said he lost the key and we had to fire him"`

Reply:
I meant the help message :)

Reply:
ah - i bet you can just copy the one as-is